library/src/main/scala/scala/scalajs/js/UndefOr.scala

```diff
@@ -199,6 +200,7 @@ final class UndefOrOps[A](val self: UndefOr[A]) extends AnyVal {
   @inline final def collect[B](pf: PartialFunction[A, B]): UndefOr[B] =
     if (isEmpty) undefined
     else pf.applyOrElse(this.forceGet, (_: A) => undefined).asInstanceOf[UndefOr[B]]
+  // scalafmt: { maxColumn = 80 }
```

Comment:
I made a pretty honest trial at fitting this under 80 characters, and it ended up *much* much worse. That's why a soft limit is important: it is important to be able to escape the length limit if the alternative is significantly worse.